api-reference/beta/resources/webhooks.md

```diff
@@ -35,22 +52,15 @@ Creating a subscription is the first step to start receiving notifications for a
 
 Client must store the subscription ID to correlate a notification with the corresponding subscription.
 
-## Characteristics of subscriptions
-
-You can create subscriptions for resources such as messages, events, contacts, and drive root items.
-
-You can create a subscription to a specific folder:
-`https://graph.microsoft.com/beta/me/mailfolders('inbox')/messages`
-
-Or to a top-level resource:
-`https://graph.microsoft.com/beta/me/messages`
-
-Or on a drive root item:
-`https://graph.microsoft.com/beta/me/drive/root`
+The following table lists the types of permissions your app can request in order to use webhooks for specific resource types. 
```

Comment:
Should we move this lead-in sentence and the table to above the "Create a subscription" section? The same permissions are required to do create, as well as renew and delete.